pkg/controller/bucketclass/bucketclass_test.go

```diff
@@ -0,0 +1,72 @@
+package bucketclass
+
+import (
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Verify Bucketclass provisioner actions", func() {
+	Context("When bucketclass is in the same namespace as NooBaa system", func() {
+		It("should allow object for the provisioner", func() {
+			systemNS := "test"
```

Comment:
maybe worth considering declaring this var as a global var outside the test case, instead of declaring in every test case, I also see it holds the same value in any case.

Reply:
Got it 👍